js/components/open-list-nav.js

```diff
@@ -126,11 +126,13 @@
 
             if (!document.getElementById(item.dataset.navItem) && item) {
               item.setAttribute('aria-hidden', true);
+              item.outerHTML = '';
+              delete item;
+            } else {
+              item.addEventListener('click', function () {
```

Comment:
Seriously @shawnbot - why would you do that! Never…ever…make functions within a loop.

:wink: 🙃